addons/web/static/src/js/core/utils.js

```diff
@@ -527,6 +527,9 @@ var utils = {
     /**
      * Patch a class and return a function that remove the patch
      * when called.
+     * 
+     * This function should not be used in intern.
+     * Use web.PatchMixin instead.
```

Comment:
This should be in the other commit.

Also, could you re-phrase like this:
"This function is the last resort solution for monkey-patching an ES6 Class, for people that do not control the code defining the Class to patch (e.g. partners), and when that Class isn't patchable already (i.e. when it doesn't have a 'patch' function, defined by the 'web.PatchMixin')."